src/services/sanity.js

```diff
@@ -254,23 +260,32 @@ export async function fetchWorkouts(brand) {
 */
 export async function fetchNewReleases(brand, { page = 1, limit = 10, sort="-published_on" } = {}) {
   const newTypes = getNewReleasesTypes(brand);
-  const typesString = arrayJoinWithQuotes(newTypes);
+  const typesString = arrayToStringRepresentation(newTypes);
   const start = (page - 1) * limit;
   const end = start + limit;
   const sortOrder = getSortOrder(sort);
-  const query = `*[_type in [${typesString}] && brand == '${brand}']{
-        "id": railcontent_id,
-        title,
-        "image": thumbnail.asset->url,
-        "artist_name": instructor[0]->name,
-        "artists": instructor[]->name,
-        difficulty,
-        difficulty_string,
-        length_in_seconds,
-        published_on,
-        "type": _type,
-        web_url_path,
-      } | order(${sortOrder})[${start}..${end}]`
+  const filter = `_type in ${typesString} && brand == '${brand}'`;
+  const fields = `
+     "id": railcontent_id,
+      title,
+      "image": thumbnail.asset->url,
+      "artist_name": instructor[0]->name,
+      "artists": instructor[]->name,
+      difficulty,
+      difficulty_string,
+      length_in_seconds,
+      published_on,
+      "type": _type,
+      web_url_path,`;
+  const filterParams = {};
+  const query = buildQuery(
+      filter,
+      filterParams,
+      fields,
+      {
+          sortOrder: 'published_on desc',
+          end:5,
```

Comment:
Shouldn't these use the variables?
```suggestion
          sortOrder: sortOrder,
          end: end,
```